api/server/flexvolume.go

```diff
@@ -1,27 +1,145 @@
 package server
 
-type flexVolumeClient struct{}
+import (
+	"fmt"
+
+	"github.com/libopenstorage/openstorage/pkg/mount"
+	"github.com/libopenstorage/openstorage/volume"
+
+	"go.pedge.io/dlog"
+)
+
+const (
+	osdDriverKey = "osdDriver"
+	volumeIdKey  = "volumeID"
+	pxDriverKey  = "pxd"
+)
+
+var (
+	deviceDriverMap map[string]string
+)
+
+type flexVolumeClient struct {
```

Comment:
one line, see near the bottom of the style guide https://github.com/libopenstorage/openstorage/blob/master/STYLEGUIDE.md